packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -468,11 +469,22 @@ class _PagePosition extends ScrollPositionWithSingleContext implements PageMetri
 ///  * [PageView.physics], which can override the physics used by a page view.
 class PageScrollPhysics extends ScrollPhysics {
   /// Creates physics for a [PageView].
-  const PageScrollPhysics({ ScrollPhysics parent }) : super(parent: parent);
+  ///
+  /// The [allowImplicitScrolling] parameter must not be null. If true, the
+  /// physics will cause the scrollable to participate in accessibility
```

Comment:
Accessibility scrolling (with three fingers on iOS or two fingers on Android) works without this setting. It's the implicit a11y scrolling (where you move the focus beyond the edge of the viewport via linear navigation) that doesn't work without this.

Reply:
The technical definition of what this really does is at https://master-api.flutter.dev/flutter/widgets/ScrollPhysics/allowImplicitScrolling.html. It is kinda independent of a11y, a11y just uses showOnScreen to do its thing.